<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>荣耀中箱AI检测系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="currentLog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>1131</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>实时日志</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="showLogResult">
       <property name="text">
        <string>实时现实日志文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="logQHBpxLayout">
     <item>
      <widget class="QLabel" name="logLevel">
       <property name="text">
        <string>日志级别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logComboBox">
       <item>
        <property name="text">
         <string>info</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>warning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>error</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1131</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="videoQGroupBox">
     <property name="title">
      <string>视频检测展示</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCameraViewfinder" name="showVideo" native="true">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="tableQGroupBox">
     <property name="title">
      <string>检测结果</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>7</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>8</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>9</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>10</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>11</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>12</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>物体</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>结果</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>当前结果</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询详情</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前检测的中箱流程为：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStartCamera"/>
   <addaction name="actStopCamera"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStartCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/video_chat.ico</normaloff>:/icons/images/video_chat.ico</iconset>
   </property>
   <property name="text">
    <string>开启摄像头</string>
   </property>
   <property name="toolTip">
    <string>开启摄像头</string>
   </property>
  </action>
  <action name="actStopCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/remove_user.ico</normaloff>:/icons/images/remove_user.ico</iconset>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
   <property name="toolTip">
    <string>关闭摄像头</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/exit_24.ico</normaloff>:/icons/images/exit_24.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>QCameraViewfinder</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
